packages/cli/internal/pkg/cli/project_init.go

```diff
@@ -125,7 +125,7 @@ func (o *initProjectOpts) validateProject() error {
 func BuildProjectInitCommand() *cobra.Command {
 	vars := initProjectVars{}
 	cmd := &cobra.Command{
-		Use:   "init project_name",
+		Use:   "init project_name --workflow-type my_workflow_type",
```

Comment:
Workflow type is effectively an enum, so `my_workflow_type` may be misleading to customers since it suggests a freetext field. I'm not sure if this is valid CLI-ese but thoughts on `"init project_name --workflow-type {wdl|nextflow}"`

The Cobra use documentation touches on this a little, but I'm not sure what it considers an argument:

```
//   { } delimits a set of mutually exclusive arguments when one of the arguments is required. If the arguments are
//       optional, they are enclosed in brackets ([ ]).
```
[source](https://pkg.go.dev/github.com/spf13/cobra@v1.2.1#Command.Use)